com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -61,28 +61,10 @@ public class NetworkConfig
         internal Dictionary<uint, NetworkPrefab> NetworkPrefabOverrideLinks = new Dictionary<uint, NetworkPrefab>();
 
         /// <summary>
-        /// Amount of times per second the receive queue is emptied and all messages inside are processed.
+        /// The tickrate of network ticks. This value controls how often MLAPI runs user code and sends out data.
         /// </summary>
-        [Tooltip("The amount of times per second the receive queue is emptied from pending incoming messages")]
-        public int ReceiveTickrate = 64;
-
-        /// <summary>
-        /// Duration in seconds between network ticks.
-        /// </summary>
-        [Tooltip("Duration in seconds between network ticks")]
-        public float NetworkTickIntervalSec = 0.050f;
-
-        /// <summary>
-        /// The max amount of messages to process per ReceiveTickrate. This is to prevent flooding.
-        /// </summary>
-        [Tooltip("The maximum amount of Receive events to poll per Receive tick. This is to prevent flooding and freezing on the server")]
-        public int MaxReceiveEventsPerTickRate = 500;
-
-        /// <summary>
-        /// The amount of times per second internal frame events will occur, e.g. send checking.
-        /// </summary>
-        [Tooltip("The amount of times per second the internal event loop will run. This includes for example NetworkVariable checking.")]
-        public int EventTickrate = 64;
+        [Tooltip("The tickrate. This value controls how often MLAPI runs user code and sends out data. The value is in 'ticks per seconds' which means a value of 50 will result in 50 ticks being executed per second or a fixed delta time of 0.02.")]
+        public int TickRate = 30;
```

Comment:
Are we confident an `int` will always do the job ? Let's say some game wanted 30 ms fixed tick rate, for [unspecified reasons], they'd be shooting for 33.33... TickRate. Pie-in-the-sky, there would be a way to specify either an integral tick per second or an integral tick duration in ms, but a more immediate fix is to offer the TickRate as a floating-point value.
(mostly asking, not really arguing for)

Reply:
Not confident. But floating point tickrate introduces some potential problems. DOTS Netcode also only support integral tickrate. MLAPI also only supported integral tickrate before. I will note it down and mention it to UX.